### ISSUE TITLE
phpcbf installed, grayed-out in menu

### ISSUE THREAD

**Issue by matthewspencer** [benmatselby/sublime-phpcs#129]
Using 6.6.0, I am unable to run the PHP Code Beautifier and Fixer in Sublime Text 3. phpcbf is installed, but the menu item is grayed-out.

Below is a screenshot of the flyout menu:

![image](https://cloud.githubusercontent.com/assets/97460/7013125/b93d419e-dc84-11e4-9caa-65dc1bd5fa82.png)

These are my user settings, confirmed paths are correct:

```json
{
	"phpcs_executable_path": "/usr/local/bin/phpcs",
	"phpcs_additional_args": {
		"--standard": "WordPress",
		"-n": ""
	},
	"phpcbf_executable_path": "/usr/local/bin/phpcbf",
	"phpcbf_additional_args": {
		"--standard": "WordPress",
		"-n": ""
	}
}
```

**Questions:**
- Is there an additional step to enable `phpcbf`?
- Is `php-cs-fixer` required for `phpcbf`?

Thanks in advance for any help!

**Comment by benmatselby** [comment]
Good spot, it's a bug on line 705:

https://github.com/benmatselby/sublime-phpcs/blob/master/phpcs.py#L705

Will get a fix soon for you, in the mean time you could just define ```php_cs_fixer_executable_path```

**Comment by matthewspencer** [comment]
@benmatselby Will do, thanks!

**Comment by benmatselby** [comment]
This should now be resolved in v6.6.1 @matthewspencer 

I have to show it as enabled as I don't have access to which tool you want to use for the menu, so now I display an error if you cannot use that tool.

Not great I have to say, but a workable solution

**Comment by matthewspencer** [comment]
@benmatselby Thank you!
